homeassistant/components/insteon/api/device.py

```diff
@@ -0,0 +1,77 @@
+"""API interface to get an Insteon device."""
+
+from pyinsteon import devices
+import voluptuous as vol
+
+from homeassistant.components import websocket_api
+
+from ..const import DOMAIN
+
+TYPE = "type"
+ID = "id"
+DEVICE_ID = "device_id"
+DEVICE_ADDRESS = "device_address"
+ALDB_RECORD = "record"
+PROPERTY_NAME = "name"
+PROPERTY_VALUE = "value"
+HA_DEVICE_NOT_FOUND = "ha_device_not_found"
+INSTEON_DEVICE_NOT_FOUND = "insteon_device_not_found"
```

Comment:
Any of these constants that are defined identically in multiple files should be added to a `const` file and then imported. `TYPE`, `ID`, `PROPERTY_NAME`, `PROPERTY_VALUE`, etc